c++/src/Statistics.hh

```diff
@@ -64,103 +64,63 @@ namespace orc {
       hasMaximum_ = false;
       hasSum_ = false;
       hasTotalLength_ = false;
-      totalLength_ = -1;
-      valueCount_ = -1;
+      totalLength_ = 0;
+      valueCount_ = 0;
     }
 
     ~InternalStatisticsImpl() {}
 
     // GET / SET totalLength_
-    bool hasTotalLength() const {
-      return hasTotalLength_;
-    }
+    bool hasTotalLength() const {return hasTotalLength_;}
```

Comment:
Spaces need to be used here to separate braces from implementation. [Horizontal_Whitespace](https://google.github.io/styleguide/cppguide.html#Horizontal_Whitespace)